### PR TITLE
clean up (link and image checking) for Accuracy, Achievements, BanchoBot, and Beatmap_Editor

### PR DESCRIPTION
**Pull request by MegaApplePi**
Inspecting all links and images; will convert `.png`s to `.jpg`s and/or downscale if need be.

---

Changes:
- `Beatmap_Editor`
  - `AiMod`
    - converted `.png` to `.jpg`
  - `Compose`
    - move images to `img` folder
    - converted `.png` to `.jpg`
    - relink images
  - `Design`
    - move images to `img` folder
    - converted `.png` to `.jpg`
    - relink images
  - `Distance_Snap`
    - move images to `img` folder
    - converted `.png` to `.jpg`
    - relink images
  - `Kiai_Time`
    - move images to `img` folder
    - converted `.png` to `.jpg`
    - relink images
  - `Menu`
    - move images to `img` folder
    - converted `.png` to `.jpg`
    - relink images
  - `SB_Load`
    - http to https
  - `Slider`
    - move images to `img` folder
    - converted `.png` to `.jpg`
    - relink images
  - `Song_Selection`
    - http to https
    - move images to `img` folder
    - converted `.png` to `.jpg`
    - relink images
  - `Timelines`
    - move images to `img` folder
    - converted `.png` to `.jpg`
    - relink images
  - `Timing`
    - move images to `img` folder
    - converted `.png` to `.jpg`
    - relink images
- `BBCode`
  - see #220 
- `BanchoBot`
  - converted `.png` to `.jpg`
  - relink image
  - fix level heading 1 abuse
  - beautify tables
- `Announcements`
  - see #219 
- `Achievements`
  - purge `it.md` -- contained much of English text
  - optimize images
  - renamed a directory
  - removed linked images
  - renamed `Beatmap Packs Achievements` to `Beatmap Packs`
- `Accuracy`
  - deleted larger file versions (not needed)
  - renamed and moved images to `img/` directory
  - relinked

### CODE REVIEW

File: wiki/BanchoBot/de.md [hunk 1]
```diff
@@ -1,37 +1,31 @@
...
-| STATS (Benutzer) | Zeigt den Status des Spielers an (basierend auf den Panel des Spielers) | !stats John | Stats for John is Idle. Score:00 (#0). Plays:2 (lvl 4). Accuracy:0.00%.
-| FAQ (L.code)(item) (list) | Verschiedene Verwendung. [list] Liste mit allen benutzbaren Befehlen | !faq wiki / !faq ru:wiki | The osu! Wiki - Make it awesome! / Примите участие в заполнении вики! |
-| REPORT (Grund) | Ruf einen Moderator | !report Thomas offending comments | Chat moderators has been alerted. Thanks for your help. |
-| REQUEST (list) | Zeigt zufällig eines der vorherigen Mod-Anfragen an. [list] Zeigt 5 zufällige, vorherig getätigte Mod-Anfragen an | !request | Seether - Fake It by [Dellirium] |
-| ROLL (Zahl) | Rollt einen (virtuellen) Würfel und gibt einen zufälligen Wert zwischen 1 und <Zahl> (Standardmäßig 100) | !roll 9000 | John rolls 1337 point(s) |
+| Befehl                    | Effekt                                                                                                            | Beispiel                          | BanchoBots Rückmeldung                                                  |
```
Comment (TPGPL) [inline wiki/BanchoBot/de.md]: why

File: wiki/Beatmap_Editor/Compose/en.md [hunk 2]
```diff
@@ -115,7 +116,7 @@ Hold `Alt` to trigger Distance Snap. The value can range from x0.1 - x2.0.
 
 **List of available subtools complement to the original DIY toolkit.**
 
-![Hit Sounds & Assist Tools](BE_HSAT.jpg "Hit Sounds & Assist Tools")
+![Hit Sounds & Assist Tools](img/BE_HSAT.jpg "Hit Sounds & Assist Tools")
```
Comment (TPGPL) [inline wiki/Beatmap_Editor/Compose/en.md]: be careful that some images are used in ~2 articles

File: wiki/BanchoBot/de.md (hunk 1 above)
Comment (MegaApplePi) [inline wiki/BanchoBot/de.md]: 👎 what

File: wiki/Beatmap_Editor/Compose/en.md (hunk 2 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Compose/en.md]: ...? this should be in `shared`?

File: wiki/Achievements/it.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Obiettivi
 Beatmap Packs
```
Comment (RaikireHiuduo) [inline wiki/Achievements/it.md]: Delete this file.

Too many English text.

File: wiki/BanchoBot/de.md (hunk 1 above)
Comment (RaikireHiuduo) [inline wiki/BanchoBot/de.md]: Too spacey for me (too much whitespace used for the whole table).

File: wiki/Achievements/it.md (hunk 3 above)
Comment (MegaApplePi) [inline wiki/Achievements/it.md]: alrighty, I was only focusing on image/links rather than contents for the most parts :/

**Pull request merged**